go.mod

```diff
@@ -4,7 +4,13 @@ go 1.17
 
 require (
 	github.com/pkg/errors v0.9.1
+	github.com/redis/go-redis/v9 v9.0.2
```

Comment:
create new module in storages/redis, not everyone will use this dependency to place it here

Reply:
resolved in #4 https://github.com/vitaliy-ukiru/fsm-telebot/pull/4/commits/90053aaa09a7426ec1ead39c4ee0fc7e433b8ae7